<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editprofile</class>
 <widget class="QMainWindow" name="editprofile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F5DEB3;
border:8px solid;
border-radius:5px;
border-color:#A0522D;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_edt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding-left:8px;
border-radius:10px;
color:#00000;
background-color:#FFF8DC;
font-size:17px;</string>
    </property>
    <property name="placeholderText">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_edt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding-left:8px;
border-radius:10px;
color:#00000;
background-color:#FFF8DC;
font-size:17px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_edt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding-left:8px;
border-radius:10px;
color:#00000;
background-color:#FFF8DC;
font-size:17px;</string>
    </property>
    <property name="placeholderText">
     <string>PhoneNumber</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>-100</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/label.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>55</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Rockwell Condensed&quot;;
color:#fff;</string>
    </property>
    <property name="text">
     <string>PROFILE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="change_btn">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>310</y>
      <width>191</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
color:#fff;
font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color:#2E8B57;
}
QPushButton:hover{
background-color:#3CB371;
}

</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/label.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>username_edt</zorder>
   <zorder>password_edt</zorder>
   <zorder>phone_edt</zorder>
   <zorder>label_2</zorder>
   <zorder>change_btn</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
